python/Ganga/GPIDev/Lib/Config/Config.py

```diff
@@ -267,14 +267,14 @@ def config_file_as_text(interactive):
                         except AttributeError as err:
                             pass
                     if interactive:
-                        yes = raw_input('The config option %s %s with value %s in your old .gangarc is not the default. Do you want to copy it to the new .gangarc file (y/[n]) ?\n' % (sect.name, o, value))
+                        yes = raw_input('The config option %s %s with value %s in your old .gangarc is not the default. Do you want to copy it to the new .gangarc file ([y]/n) ?\n' % (sect.name, o, value))
                     else:
                         yes = 'y'
-                    if yes.lower() in ['y']:
+                    if yes.lower() in ['n']:
                         text += '#%s = %s\n' % (o, def_value)
-                        text += '%s = %s\n\n' % (o, value)
                     else:
                         text += '#%s = %s\n' % (o, def_value)
+                        text += '%s = %s\n\n' % (o, value)
```

Comment:
Nothing really to do with the specific changes (which are fine!), but this could actually be condensed to:
```
text += '#%s = %s\n' % (o, def_value)
if yes.lower() not in ['n']:
    text += '%s = %s\n\n' % (o, value)
```

Knocks a couple of lines off the file 😄